frontend/src/app/page.tsx

```diff
@@ -23,13 +23,18 @@ export default function WebSocketPage() {
       console.log("Connected to WebSocket server");
 
       try {
-        const location = await getCurrentPosition();
-        const latitude = location?.coords.latitude;
-        const longitude = location?.coords.longitude;
-        socket.emit("init", { latitude, longitude }); // 'init' イベントで送信
-        console.log(
-          "Sent location data to server : " + { latitude, longitude }
-        );
+        const currentPosition = await getCurrentPosition();
+        const latitude = currentPosition?.coords.latitude;
+        const longitude = currentPosition?.coords.longitude;
+
+        if (!latitude || !longitude) {
+          throw new Error("Failed to get location data");
+        }
+
+        const name = "noname";
+        const position = { latitude, longitude };
+        socket.emit("init", { name, position }); // 'init' イベントで送信
+        console.log("Sent location data to server : " + currentPosition);
```

Comment:
現状が
```
Sent location data to server : [object GeolocationPosition]
```
これなので
```suggestion
        console.log("Sent location data to server : ", currentPosition?.coords.latitude, currentPosition?.coords.longitude);
```
こんなんでどうでしょう